app/scripts/templates/settings/two_step_authentication.mustache

```diff
@@ -59,7 +59,7 @@
         <form novalidate>
             <div id="totp" class="totp-details hidden">
                 <div class="qr-image-container">
-                    <img class="qr-image"/>
+                    <img class="qr-image" alt="Two step authentication QR code image"/>
```

Comment:
Open to suggestions for what might be better here.

Reply:
Does this have to be translated?